lib/mix/tasks/ecto.drop.ex

```diff
@@ -28,6 +28,18 @@ defmodule Mix.Tasks.Ecto.Drop do
 
     {opts, _, _} = OptionParser.parse args, switches: [quiet: :boolean]
 
+    if disable_safety_warnings?(repo) or Mix.shell.yes?("Are you sure you want to drop the database for repo #{inspect repo}?") do
```

Comment:
If I compare with `true` we have the same problem, right? Configuration like:

```elixir
disable_safety_warnings: "true"
# or
disable_safety_warnings: "false"
```

are the same. It's documented that this option is a boolean, so I think we're safe?